src/Pos/Txp/Toil/Logic.hs

```diff
@@ -109,55 +76,32 @@ rollbackToil txun = do
 type LocalTxpMode m = ( MonadUtxo m
```

Comment:
Why `GlobalToilMode`, but `LocalTxpMode`? Isn't it intended to be `LocalToilMode`?